app/views/foods/general_shopping_list.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="shopping_list" id="shopping">
+  <h1>Shopping List</h1>
+
+  <% if @missing_food_items.any? %>
+    <div class="total">
+      <p>Amount of food to buy: <%= @total_food_items %></p>
+      <p>Total value of food need: <%= @total_price %></p>
+    </div>
+    <div class="table">
+      <table>
+        <thead>
+          <tr>
+            <th>Food </th>
+            <th>Quantity</th>
+            <th>Price</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @missing_food_items.each do |item| %>
+            <tr>
+              <td><%= item[:food_name] %></td>
+              <td> <%= item[:quantity_needed] %></td>
+              <td> $<%= item[:price] %></td>
+            </tr>
+          <% end %>
+        </tbody>
+      </table>
+    </div>
+  <% else %>
+    <p>No missing food items.</p>
+  </div>
+  <% end %>
+  <%= link_to 'Add Food', new_food_path, class: "add-button" %>
+</div>
```

Comment:
- [ ] As highlighted by @abdulhamiid, it is important to make your designs identical to wireframes provided in an application. Implementing designs based on wireframes ensures a consistent and accurate translation of the intended user interface, fostering alignment between design expectations and the final product. Kindly make the **_General Shopping List_** page look identical to the wireframe provided. 💻 👍 

### Your Design

![image](https://github.com/Gardimy/ror_recipe_app/assets/77106962/ce0128c8-b18c-46e3-818e-d196d7d24e3e)

### Wireframe Provided

![image](https://github.com/Gardimy/ror_recipe_app/assets/77106962/dd1b1e25-f9b7-483b-a54e-b85a8be0fd55)

#### N.B: Please note that the **_Add Food_** button should not be on this page. 😃